src/DependencyInjection/Configuration.php

```diff
@@ -28,6 +28,10 @@ private function addConverterSection(ArrayNodeDefinition $rootNode): void
     {
         $rootNode
             ->children()
+                ->scalarNode('converterSuffix')
```

Comment:
Config parameters should be in "snake_case":

```suggestion
                ->scalarNode('converter_suffix')
```
<br>
Maybe we should even rename this to be more specific, like `argument_autowiring_alias_suffix`.